ceremonies/src/lib.rs

```diff
@@ -1266,37 +1262,51 @@ impl<T: Config> Pallet<T> {
 
 		info!(target: LOG, "purging ceremony index {} history for {:?}", cindex, cid);
 
+		#[allow(deprecated)]
```

Comment:
You could set this flag at the fn definition and it should be calid for the entire fn. Or do yiu want to isolate steictly?

Reply:
indeed this is nicer!